modules/serverpod_auth/serverpod_auth_email_flutter/lib/src/signin_dialog.dart

```diff
@@ -510,9 +510,9 @@ class SignInWithEmailDialogState extends State<SignInWithEmailDialog> {
 
   Future<void> _resetPassword() async {
     _resetIssues();
-    if (_passwordController.text.length < 8) {
+    if (_passwordController.text.length < widget.minPasswordLength) {
       setState(() {
-        _passwordIssue = 'Min 8 characters';
+        _passwordIssue = 'Min ${widget.minPasswordLength} characters';
```

Comment:
No need to change but this change should be tagged as `fix` instead of `chore`, chore normally refers to none source code changes such as dependency updates or changes in config files.

Reply:
Aah thanks! 🙏